app/(pages)/ecommerce/_components/EcommercePricingPlansSection/EcommercePricingPeriodToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from "react";
+
+import { ToggleButton } from "components";
+
+type TProps = {
+  activePeriod: string;
```

Comment:
Should be something different than a string, it's 2 specified values, we use string if we know that would be a random string like user input, if we know that the values will be hardcoded we should use a pipe operator `'this string' | 
'that string'`

If there is a lot values we should use enums